docs/going-into-production.rst

```diff
@@ -0,0 +1,357 @@
+:ref:`bootstrap checks <bootstrap-checks>`. These checks may require changes to
+your operating system configuration.
+
+.. SEEALSO::
+
+    `Host settings`_
+
+
+.. _prod-config-paths:
+
+Configure persistent data paths
+===============================
+
+By default, CrateDB keeps data under the `CRATE_HOME`_ directory (which
+defaults to the installation directory). When you upgrade CrateDB, you will
+have to switch to a new installation directory.
+
+Instead of migrating data by hand each time, you should move the data
+directories off to a persistent location. You can do this using the
+`CRATE_HOME`_ environment variable and the `path settings`_ in your
+`configuration`_ file.
+
+For example, if you are running CrateDB on a `Unix-like`_ operating system, the
+`Filesystem Hierarchy Standard`_ recommends the ``/srv`` directory as the root
+for site-specific data.
+
+With this in mind, a good value for `CRATE_HOME`_ on a Unix-like system might
+be ``/srv/crate``. Make sure to set `CRATE_HOME` before running `bin/crate`_.
+
+Then, you could configure your data paths like this:
+
+.. code-block:: yaml
+
+    path.conf: /srv/crate/config
+    path.data: /srv/crate/data
+    path.logs: /srv/crate/logs
+    path.repo: /srv/crate/snapshots
+
+
+Path configuration is especially important if you are running CrateDB on
+Docker. Persistent data should be located on a mounted volume.
+
+.. TIP::
+
+    You should take care size the data storage volume according to your needs.
+    You should also use storage with high `IOPS`_ to improve CrateDB
+    performance.
+
+.. SEEALSO::
+
+    `Path settings`_
+
+
+.. _prod-jvm:
+
+Tune the JVM
+============
+
+
+.. _prod-config-heap:
+
+Heap
+----
+
+CrateDB is a Java application running on top of a Java Virtual Machine (JVM).
+The JVM uses a heap for memory allocations. For optimal performance, you must
+pay special attention to your :ref:`heap configuration <memory>`.
+
+By default, CrateDB configures the JVM to dump out-of-memory exceptions to the
+file or directory specified by `CRATE_HEAP_DUMP_PATH`_. You must make sure
+there is enough disk space available for heap dumps at this location.
+
+.. SEEALSO::
+
+    `JVM environment variables`_
+
+
+.. _prod-config-gc:
+
+Garbage collection
+------------------
+
+CrateDB logs JVM garbage collection times using the built-in *garbage
+collection* (GC) logging provided by the JVM. You can configure this process
+with the `GC logging environment variables`_.
+
+You must ensure that the log directory is on a fast-enough disk and has enough
+space. When using Docker, use a path on a mounted volume.
+
+If garbage collection takes too long, CrateDB will log this. You can adjust the
+`timeout settings`_ to suit your needs. However, the default settings should
+work in most instances.
+
+If you are running CrateDB on Docker, you should configure the container to
+send debug logs to `STDERR`_ so that the container orchestrator handles the
+output.
+
+
```

Comment:
Still don't like the end, I get your point, but maybe we can add something like further steps linking to administration,...

Reply:
I am sticking by my guns on this. I have reviewed our collection of how-to guides. I do not see any specific document or section that makes sense as a logical follow-on. at best, we would be guessing what the reader wants to do next

the tutorials are designed to follow-on from each other. i.e., we expect users to start at the beginning and work their way through

however, the how-to guides are more like the reference. they are designed for readers to dip in and dip out as needed. we do not expect people to read them sequentially

the how-to guides and the reference are roughly ordered by how advanced the topic is and how important/central the topic is. but that does not imply a particular "flow" through the docs

Reply:
you should drop your guns